sunpy/map/mapbase.py

```diff
@@ -1010,15 +1011,32 @@ def center(self):
     def shifted_value(self):
         """The total shift applied to the reference coordinate by past applications of
         `~sunpy.map.GenericMap.shift`."""
+        warn_deprecated(
+            '`sunpy.map.GenericMap.shifted_value` is deprecated and will be removed in sunpy 4.1. '
+            'Use ``sunpy.map.GenericMap.meta.modified_items`` to see how the '
+            'reference coordinate has been modified.'
+        )
         return self._shift
 
+    @deprecated('4.0', alternative='`sunpy.map.GenericMap.shift_reference_coord`')
     @u.quantity_input
     def shift(self, axis1: u.deg, axis2: u.deg):
+        GenericMap.shift.__doc__ = GenericMap.shift_reference_coord.__doc__
```

Comment:
I don't think updating the docstring here has any effect. Should it be updated at the bottom of the file instead? [Here's it in the docs.](https://sunpy--5977.org.readthedocs.build/en/5977/generated/api/sunpy.map.GenericMap.html#sunpy.map.GenericMap.shift)

Reply:
I tried that but it doesn't update the docstring. Inside the method the docstring must be a string literal. The code inside the method isn't run unless it is called and doing `help(GenericMap.shift)` doesn't do that.

Reply:
Thanks for checking, to the end of the file it is.